docs/glossary.txt

```diff
@@ -75,3 +75,5 @@ Glossary
 
     view
         A function responsible for rendering a page.
+     App
+        A small library representing a discrete part of a larger project
```

Comment:
This term is not referenced anywhere in the docs. Moreover the differences between apps and projects are described several times in the docs, e.g.
- https://docs.djangoproject.com/en/stable/intro/tutorial01/#creating-the-polls-app
- https://docs.djangoproject.com/en/stable/intro/reusable-apps/#reusability-matters